model/README.md

```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
 surroundings. To support this, new DOM APIs may be added or the existing HTML Fullscreen API extended
 via more [FullscreenOptions](https://fullscreen.spec.whatwg.org/#dictdef-fullscreenoptions) properties.
 
+### Controlling the camera
+
+While by setting the `interactive` property the author will allow a built-in behavior such that dragging
+over a `<model>` element would result in modifying the camera, we propose to allow authors direct control
+of the camera via DOM APIs. An initial proposition would be to add an `HTMLModelElementCamera`:
+
+```idl
+dictionary HTMLModelElementCamera {
+    double pitch;
+    double yaw;
+    double scale;
+};
+```
+
+Then the camera can be set and read back:
+
+```idl
+interface HTMLModelElement : HTMLElement {
+    Promise<HTMLModelElementCamera> getCamera();
+    Promise<undefined> setCamera(HTMLModelElementCamera camera);
+}
+```
+
+Note the use of promises since it is likely that the model is rendered out-of-process and any communication
+with that process would need to be asynchronous. This applies to other promise-based APIs discussed in this
+document.
+
+### Controlling animations
+
+Formats supported by `<model>` may support animations built into the resource itself, such as the  USDZ
+file format. We propose allowing page authors to control such animations.
+
+This is a wide topic with likely dependencies on the file format support for animations itself. Another
+important topic would be whether the Web Animations specification could be leveraged to expose and control
+animations for the resource.
+
+For experimental purposes, we propose an initial, basic set of DOM APIs based on the assumption that
+a single animation is controlled. With this proposal the author could control whether the animation is
+playing, looping, query its duration and set its current time, allowing the creation of controls to
+toggle playback and scrub through the animation.
+
+```idl
+interface HTMLModelElement : HTMLElement {
```

Comment:
Might be nice to say how this fits with Web Animation timelines. Is this itself a timeline, or does it provide or subscribe to one?